workers/unity/Packages/io.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Commands/Create.cs

```diff
@@ -139,29 +137,28 @@ private static string ModifySimulatedPlayerLaunchJson(Options.CreateSimulated op
             return launchConfig.ToString();
         }
 
-        private static string UploadSnapshot(SnapshotServiceClient client, string snapshotPath, string projectName,
-            string deploymentName)
+        private static string UploadSnapshot(SnapshotServiceClient client, Options.Create options)
         {
-            if (!File.Exists(snapshotPath))
+            if (!File.Exists(options.SnapshotPath))
             {
-                Ipc.WriteError(Ipc.ErrorCode.NotFound, $"Could not find snapshot file at: {snapshotPath}");
+                Ipc.WriteError(Ipc.ErrorCode.NotFound, $"Could not find snapshot file at: {options.SnapshotPath}");
                 return null;
             }
 
             // Read snapshot.
-            var bytes = File.ReadAllBytes(snapshotPath);
+            var bytes = File.ReadAllBytes(options.SnapshotPath);
```

Comment:
catch exceptions here?

Reply:
What exceptions, this is out of scope for this PR

Reply:
rip didn't see the lines above this - ignore